esphome/components/font/__init__.py

```diff
@@ -207,6 +270,14 @@ def validate_file_shorthand(value):
             data[CONF_WEIGHT] = weight[1:]
         return FILE_SCHEMA(data)
 
+    if value.startswith("https"):
```

Comment:
unencrypted http is not fully dead yet ;-)
```suggestion
    if value.startswith("http://") or value.startswith("https://"):
```

Reply:
long live http. what do you suggest? I can look for a regex to match URLs if you think that works?